### DISCUSSION TITLE
This form is not secure | login page

### DISCUSSION THREAD

**Discussion by noor-muradi** [snipe/snipe-it#12292] [General]
Hello folks,
I have deployed SnipeIT docker using the latest docker image behind Nginx reverse proxy, everything is perfect except one issue that I am facing, which is the "The form is not secure autofill has been turned off" on login page, I am not good at coding actually as I am a junior DevOps engineer, I would glad if someone help me to fixed it.

**Comment by iangozer** [comment]
Have you configured your setup to use https? This is probably your issue.

**Reply by noor-muradi** [reply]
Thanks for the reply, I got it fixed by putting "APP_FORCE_TLS=true" in .env file.
